resources/assets/js/components/WordCounter/WordCounter.test.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  countNumberOfWords,
+  stringEndsInWhitespace,
+  truncateWords,
+} from "./WordCounter";
```

Comment:
This file is now testing the WordCounter utility file, now the component itself. Imports need to change, and probably the describe wordings.